docs/dispatcher/router.rst

```diff
@@ -12,12 +12,12 @@ Event observers
 
 .. warning::
 
-    All handlers is always should be an asynchronous.
-    Name of handler function is not important. Event argument name is also is not important but is recommended to don't overlap the name with contextual data in due to function can not accept two arguments with the same name.
+    All handlers always should be asynchronous.
+    The name of the handler function is not important. The event argument name is also not important but it is recommended to not overlap the name with contextual data in due to function can not accept two arguments with the same name.
 
-Here is list of available observers and examples how to register handlers (In examples used only @decorator-style):
+Here is the list of available observers and examples of how to register handlers (In examples used only @decorator-style):
```

Comment:
I guess it would be better to remove this repeating sentence and extra colon

```suggestion
Here is the list of available observers and examples of how to register handlers
```